src/components/Button.tsx

```diff
@@ -67,6 +67,7 @@ export default function Button({
           "&::before": {
             borderRadius: 4,
           },
+          width: '100%',
```

Comment:
@andymatuschak is this ok with you? It doesn't affect the "Return to book" / "Review later" button. But not sure if you wanted the button to be sized by its contents, in the video I noticed the save prompt button was the width of the prompts, so I modeled this off of that.

Reply:
Seems fine for now—thanks for checking in! We can fiddle with it if it causes problems elsewhere.